Author: Andreas Beckmann <anbe@debian.org>
Description: do not attempt to build the module on kernels without CONFIG_ACPI

--- a/dkms/dkms.conf
+++ b/dkms/dkms.conf
@@ -5,3 +5,4 @@ CLEAN="make KVERSION=$kernelver clean"
 BUILT_MODULE_NAME[0]="bbswitch"
 DEST_MODULE_LOCATION[0]="/kernel/drivers/acpi"
 AUTOINSTALL="yes"
+BUILD_EXCLUSIVE_CONFIG="CONFIG_ACPI"
